ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/LdesFragmentController.java

```diff
@@ -31,7 +31,7 @@ public LdesFragmentController(FragmentFetchService fragmentFetchService) {
     }
 
 
-    @GetMapping(value = "${ldes.collectionname}", produces = {"application/ld+json", "application/n-quads", "application/turtle"})
+    @GetMapping(value = "${ldes.collectionname}", produces = {"application/turtle", "application/ld+json", "application/n-quads"})
```

Comment:
forgot to update this. since we now support all formats that Jena can handle, this "produces" part should be removed

Reply:
Let's handle in separate PR.